packages/code-du-travail-frontend/src/common/ViewMore.js

```diff
@@ -3,40 +3,54 @@ import PropTypes from "prop-types";
 import styled from "styled-components";
 import { Button, FlatList, theme } from "@socialgouv/react-ui";
 
-const ViewMore = ({ children, elementsDisplayed, label, onClick, query }) => {
-  const [currentPage, setCurrentPage] = useState(1);
+const ViewMore = ({
+  buttonProps,
+  children,
+  CustomContainer,
+  elementsDisplayed,
+  label,
+  onClick,
+  query,
+}) => {
+  const [totalDisplayed, setTotalDisplayed] = useState(1);
   useEffect(() => {
-    setCurrentPage(1);
-  }, [query]);
+    setTotalDisplayed(elementsDisplayed);
```

Comment:
I don't understand this useEffect here. If the purpose is to make its behaviour different, I would probably us a différent props here like  `visibleElements` / `pageSize` to make it clearer

Reply:
Naming is hard ! I didn't like the "page" word in a component whose purpose is in fact to prevent having pages / pagination !
What about this ?
elementsDisplayed => stepSize
totalDisplayed => currentListSize